decls/haskell_common.bzl

```diff
@@ -24,6 +24,10 @@ def _deps_arg():
      from which this rules sources import modules or native linkable rules exporting symbols
      this rules sources call into.
 """),
+        "srcs_deps": attrs.dict(attrs.source(), attrs.list(attrs.source()), default = {}, doc = """
+    Allows to declare dependencies for sources manually, in case the automatic
+        dependency mechanism is not able to detect them.
```

Comment:
The way it's phrased it sounds like it would override the automatic dependency detection. But, IIUC it extends the set of automatically discovered dependencies, correct?

Reply:
How about:

Reply:
That sounds good, thanks!